apps/server/src/modules/classes/entity/class-source-options.entity.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { ClassSourceOptionsEntity } from './class-source-options.entity';
+
+describe('Class Source Options Entity', () => {
```

Comment:
You can do just _ClassSourceOptionsEntity.name_ so if the class name changes, this string will also be correct without requiring additional refactor.

Reply:
fixed